Resources/Private/Layouts/FrontEndEditor.html

```diff
@@ -0,0 +1,13 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
+    <f:security.ifAuthenticated>
+        <f:then>
+            <f:render section="main"/>
+        </f:then>
+        <f:else>
+            <div class="alert alert-danger" role="alert">
+                Please configure this plugin to be only visible if a FE user in the group with the
```

Comment:
The message says `in the group` but the check is against any authentication.
Will that change with an upcoming PR, should we change the message or the check?

Reply:
Thanks, I've updated the message now. (It was a copy'n'paste thing.)